src/index.ts

```diff
@@ -18,6 +18,7 @@ app.all("*", async (context) => {
   const xGatewayServiceToken = clone.headers.get("x-gateway-service-token");
   const xGatewayServiceAuthKey = clone.headers.get("x-gateway-service-auth-key");
   const xGatewayServiceAuthType = clone.headers.get("x-gateway-service-auth-type");
+  const xGatewayAuthorizationType = clone.headers.get("x-gateway-service-authorization-type");
```

Comment:
Yeah I think prefix instead of type makes more sense?

Reply:
ok